lib/screens/home_screen.dart

```diff
@@ -103,87 +92,30 @@ class HomeScreenContent extends StatelessWidget {
-                        padding: EdgeInsets.only(left: 10.sp),
-                        child: IconButton(
-                          onPressed: () {
-                            context.read<EaselProvider>().videoLoadingError = '';
-                            context.read<EaselProvider>().isVideoLoading = true;
-                            ScaffoldMessenger.of(context).hideCurrentSnackBar();
-                            homeViewModel.previousPage();
-                            if (homeViewModel.currentPage.value == 0) {
-                              GetIt.I.get<CreatorHubViewModel>().getDraftsList();
-                              Navigator.of(context).pop();
-                            }
-                          },
-                          icon: const Icon(
-                            Icons.arrow_back_ios,
-                            color: EaselAppTheme.kGrey,
-                          ),
-                        )),
-                  )),
-              ValueListenableBuilder(
-                valueListenable: homeViewModel.currentPage,
-                builder: (_, int currentPage, __) {
-                  return Text(
-                    homeViewModel.pageTitles[homeViewModel.currentPage.value],
-                    style: Theme.of(context).textTheme.bodyText1!.copyWith(fontSize: 18.sp, fontWeight: FontWeight.w400, color: EaselAppTheme.kDarkText),
-                  );
-                },
-              ),
-            ],
-          ),
-          ScreenResponsive(
-            mobileScreen: (context) => const VerticalSpace(6),
-            tabletScreen: (context) => const VerticalSpace(30),
-          ),
-        ],
-        Expanded(
-          child: PageView.builder(
-            controller: homeViewModel.pageController,
-            physics: const NeverScrollableScrollPhysics(),
-            onPageChanged: (int page) {
-              homeViewModel.currentPage.value = page;
-              switch (page) {
-                case 0:
-                  homeViewModel.currentStep.value = 0;
-                  break;
-                case 1:
-                case 2:
-                  homeViewModel.currentStep.value = 1;
-                  break;
-
-                case 3:
-                  homeViewModel.currentStep.value = 2;
-                  break;
-              }
-            },
-            itemBuilder: (BuildContext context, int index) {
-              final map = {0: chooseFormatScreen, 1: describeScreen, 2: priceScreen, 3: publishScreen};
-
-              return map[index]?.call() ?? const SizedBox();
-            },
-          ),
-        ),
-      ],
+    return PageView.builder(
+      controller: homeViewModel.pageController,
+      physics: const NeverScrollableScrollPhysics(),
+      onPageChanged: (int page) {
+        homeViewModel.currentPage.value = page;
+        switch (page) {
+          case 0:
+            homeViewModel.currentStep.value = 0;
+            break;
+          case 1:
```

Comment:
Is case 1 supposed to be empty?

Reply:
Yes so the stepper are 3 and the screens are five so this solves this.